src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/EditorInProcess.cs

```diff
@@ -32,24 +30,17 @@
 using Microsoft.VisualStudio.TextManager.Interop;
 using Microsoft.VisualStudio.Utilities;
 using Roslyn.Utilities;
+using Roslyn.VisualStudio.IntegrationTests.InProcess;
 using Xunit;
 using IObjectWithSite = Microsoft.VisualStudio.OLE.Interop.IObjectWithSite;
 using IOleCommandTarget = Microsoft.VisualStudio.OLE.Interop.IOleCommandTarget;
 using IOleServiceProvider = Microsoft.VisualStudio.OLE.Interop.IServiceProvider;
 using OLECMDEXECOPT = Microsoft.VisualStudio.OLE.Interop.OLECMDEXECOPT;
 
-namespace Roslyn.VisualStudio.IntegrationTests.InProcess
+namespace Microsoft.VisualStudio.Extensibility.Testing
 {
-    internal class EditorInProcess : InProcComponent
+    internal partial class EditorInProcess
```

Comment:
@sharwell Does the "InProcess" mean much here anymore, or is this just still in the process of being refactored?

Reply:
(I was under the impression it's all in-proc now, but maybe that understanding is just wrong.)

Reply:
Multiple processes are still involved. For example, if you mark a test with `[Fact]` instead of `[IdeFact]`, it will run in the test process instead of inside Visual Studio. The InProcess suffix designates types that only work when the caller is in devenv.

Reply:
It's a convention we have the option of changing.

Reply:
Ah, I had (perhaps naively) assumed everything was just the IdeFact kind. I guess I can imagine some uses of the other kind too.